packages/UsageAnalysis/files/Test Track/Viewers/Pivot table.md

```diff
@@ -1,7 +1,6 @@
 Check tickets:
 
 * [#2606](https://github.com/datagrok-ai/public/issues/2606)
-* [#2166](https://github.com/datagrok-ai/public/issues/2166)
```

Comment:
Why did we remove it? did we create a full test case for it?

Reply:
@ssSavenko FYI